pig/src/main/java/org/jboss/pnc/bacon/pig/PigFacade.java

```diff
@@ -241,34 +244,26 @@ private static void pushToBrew(boolean reimport) {
                     "Pushing the following builds to brew: {}",
                     buildsToPush.stream().map(PncBuild::getId).collect(Collectors.toList()));
         }
-        try (AdvancedBuildClient pushingClient = new AdvancedBuildClient(PncClientHelper.getPncConfiguration())) {
-            for (PncBuild build : buildsToPush) {
-                BuildPushParameters request = BuildPushParameters.builder()
-                        .tagPrefix(tagPrefix)
-                        .reimport(reimport)
-                        .build();
-
-                // TODO: customize the timeout
-                try {
-                    BuildPushResult pushResult = pushingClient
-                            .executeBrewPush(build.getId(), request, 15L, TimeUnit.MINUTES);
-                    if (pushResult.getStatus() != BuildPushStatus.SUCCESS) {
-                        throw new RuntimeException(
-                                "Failed to push build " + build.getId() + " to brew. Push result: " + pushResult);
-                    }
-                    log.info("{} pushed to brew", build.getId());
-                } catch (RemoteResourceException e) {
-                    throw new RuntimeException("Failed to push build " + build.getId() + " to brew", e);
+        for (PncBuild build : buildsToPush) {
```

Comment:
@akoufoudakis are the changes here intentional?

Reply:
Nope... I haven't change anything in this place. I think it is caused by some bad rebasing on my side.

Reply:
You can just run again: `git fetch upstream; git rebase upstream/master` and fix the conflicts.

Reply:
@akoufoudakis By the way, I tried this command and it worked for me and I got all your commits at the top. But, I'd rather you do it to make sure unless you can't figure out how.